plugin/PlayAssetDelivery/src/main/java/com/google/play/core/godot/assetpacks/PlayAssetDelivery.java

```diff
@@ -192,10 +251,12 @@ public Dictionary getPackLocations() {
    * state or details for the specified asset packs. Emits getPackStatesSuccess and
    * getPackStatesError signals when the underlying task succeeds/fails.
    *
-   * @param packNames list of name for all the packs to request states
+   * @param packNamesArray String Array for all the packs to request states
    * @param signalID identifier used to track mapping of signals to Tasks
    */
-  public void getPackStates(List<String> packNames, int signalID) {
+  public void getPackStates(final String[] packNamesArray, int signalID) {
```

Comment:
Fixed a bug here, since Godot runtime uses String Array to pass in data. See [here](https://github.com/godotengine/godot-google-play-billing/blob/916385d3ca2dbbff493fb53272921b1ca8d71e65/godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/GodotGooglePlayBilling.java#L105).